content/post/Dabbling_with_react_@_React_India_2019.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Dabbling with React @ React India 2019: A Tweet story"
+slug = "reactindia2019-vipulgupta2048"
+image = "images/react1.png"
+date = "2019-10-07T10:10:00"
+description = "My experience of India's first React conf, React India 2019"
+disableComments = false
+author = "Vipul Gupta"
+categories = ["conferences"]
+tags = ['React', 'GraphQL', 'javascript',  'vipulgupta2048', 'community', 'FOSS', 'Redux', 'India', 'React-Native', 'conferences']
++++
+
+Lush green trees, monsoon weather, soul cooling winds coming from the tumoultous waters of the beach where sounds of waves splashing over the dry sand can be heard by people walking by. Now, imagine a tech conference on a piece of software that today powers almost half of the web at this place that I described. How can you be able to stop yourself? I couldn't.
+
+React India was the first ever international, community-led beach conference that provided a platform for developers to share and discuss their insights and experiences with React. The three-day conference is the first of its kind in India broken into workshops and conference days with community talks from 26th September - 28th September 2019
+
+Before leaving for the conference, I decided to do something for the first time. It was to capture a picture of everyone I met at the conference and tweet about it, that way when the conference ends. I can write this blog much more easily. Now, you know why its a tweet story. 
+
+## The Journey 
+
+The journey starts with me filling the diversity scholarship form for React India and in turn getting selected from a pool of hundreds. Happy to say, 9 other people who were selected were gems in their own fields as I had a marvellous time interacting with each and everyone.
+
+Hopped on the early morning flight to Goa and was met with brief patch of Goa rains with stormy weather. As we travelled to the South side of Goa with the bus filled of React India attendees.
```

Comment:
```suggestion
I hopped on the early morning flight to Goa and was met with a brief patch of Goa rains with stormy weather. As we travelled to the South side of Goa with the bus filled with React India attendees.
```

Reply:
were the rains meant for the flight or the bus trip. 
'cause the last sentence seems incomplete. As we 'did something' ....?

Reply:
Completed it, good catch.